packages/bot-web-ui/src/components/dashboard/bot-builder/toolbox/toolbox.tsx

```diff
@@ -0,0 +1,165 @@
+                                                className='db-toolbox__item'
+                                                onClick={() => {
+                                                    // eslint-disable-next-line no-unused-expressions
+                                                    has_sub_category
+                                                        ? onToolboxItemExpand(index)
+                                                        : onToolboxItemClick(category);
+                                                }}
+                                            >
+                                                <div className='db-toolbox__category-text'>
+                                                    <div className='db-toolbox__label'>
+                                                        {localize(category.getAttribute('name'))}
+                                                    </div>
+                                                    {has_sub_category && (
+                                                        <div
+                                                            className={classNames('db-toolbox__category-arrow', {
+                                                                'db-toolbox__category-arrow--active':
+                                                                    is_sub_category_open,
+                                                            })}
+                                                        >
+                                                            <Icon icon='IcChevronDownBold' />
+                                                        </div>
+                                                    )}
+                                                </div>
+                                            </div>
+                                            {has_sub_category &&
+                                                is_sub_category_open &&
+                                                (Array.from(category.childNodes) as HTMLElement[]).map(subCategory => {
+                                                    return (
+                                                        <div
+                                                            key={`db-toolbox__sub-category-row--${subCategory.getAttribute(
+                                                                'id'
+                                                            )}`}
+                                                            className='db-toolbox__sub-category-row'
+                                                            onClick={() => {
+                                                                onToolboxItemClick(subCategory);
+                                                            }}
+                                                        >
+                                                            <Text size='xxs' line_height='m'>
```

Comment:
line_height='m' is default for Text component, no need to pass it here

Reply:
done ✅